spec/thread_safety_spec.rb

```diff
@@ -31,13 +31,15 @@ def current_thread_id(#{args_str})
     # which caused accidental `nil` values to be returned under contended calls
     # to unmemoized zero-args methods.
     #   * See https://github.com/panorama-ed/memo_wise/pull/224
-    context "when checking condition: are nil values accidentally returned?" do
-      let(:condition_to_check) do
-        ->(values) { values.any?(&:nil?) }
-      end
+    unless RUBY_PLATFORM == "java"
```

Comment:
Do we not want to run these on MRI? Can you add some context in a comment?

Reply:
Added some context, and #264. Let me know how it looks now